client/gutenberg/extensions/mailchimp/view.scss

```diff
@@ -1,49 +1,32 @@
 .wp-block-jetpack-mailchimp {
+
 	&.is-processing {
 		form {
 			display: none;
 		}
 	}
-	.wp-block-jetpack-mailchimp-notification {
+
+	.wp-block-jetpack-mailchimp_notification {
 		display: none;
-		margin-bottom: 23px;
-		padding: 20px;
-		text-align: center;
+		margin-bottom: 1.5em;
+		padding: 0.75em;
 		&.is-visible {
 			display: block;
 		}
+
+		&.wp-block-jetpack-mailchimp_error {
+			background-color: #eb0001;
+			color: #fff;
+		}
+
+		&.wp-block-jetpack-mailchimp_processing {
+			background-color: rgba( 0, 0, 0, 0.025 );
+		}
+
+		&.wp-block-jetpack-mailchimp_success {
+			background-color: #1d7819;
```

Comment:
We could use `$muriel-green-500` here.

Reply:
Since we are using Hot Red and I didn't think of it previously, let's use Hot Green here instead 😊 `$muriel-hot-red-500`

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/30685/commits/83bc6ab8cac6a70717947972975d00b768781b52